npm/pkg/dataplane/policies/policy.go

```diff
@@ -0,0 +1,77 @@
+package policies
+
+import (
+	"github.com/Azure/azure-container-networking/npm/pkg/dataplane/ipsets"
+	networkingv1 "k8s.io/api/networking/v1"
+)
+
+type NPMNetworkPolicy struct {
+	Name string
+	// PodSelectorIPSets holds all the IPSets generated from Pod Selector
+	PodSelectorIPSets []*ipsets.IPSet
+	// OtherIPSets holds all IPSets generated from policy
+	// except for pod selector IPSets
+	OtherIPSets []*ipsets.IPSet
+	ACLs        []*ACLPolicy
+	// Making this a podKey instead should be
+	// use NPMPod obj
+	Pods  []string
+	RawNP *networkingv1.NetworkPolicy
+}
+
+type ACLPolicy struct { // Iptable rules
+	// PolicyID is the rules name with a given network policy
+	PolicyID string
+	// Comment is the string attached to rule to identity its representation
```

Comment:
Does windows stack also support `comment`? 
Even I think comment will be generated in each dataplane?

Reply:
Comment is not useful today in windows dataplane and comment is generated in translate policy. I left this in for debugging purposes in case of windows.